metrics_handler/README.md

```diff
@@ -0,0 +1,132 @@
+# Metrics Handler
+
+The metrics handler runs as a Cloud Function that is kicked off periodically by a Cloud Scheduler.
+
+Each time it runs, it checks for tests that have finished running and collects the final status of each test plus any training metrics that the job wrote to Tensorboard. It writes the job status into a BigQuery table called `job_history` and writes the metrics into a BigQuery table called `metric_history`.
+
+
+## Setup
+
+Once you have cloned the repo, create the Cloud Function + Cloud Scheduler with
+these commands:
+
+1. `cd metrics_handler`
+2. `gcloud functions deploy metrics_handler --runtime python37 --trigger-topic=begin-metrics-handler --memory=1024MB --entry-point=run_main --timeout=500s`
+3. `gcloud scheduler jobs create pubsub metrics-handler --schedule="*/15 * * * *" --topic=begin-metrics-handler --message-body="{}" --description="Kicks off the metric handler"`
+
+
+## Config
+
+The behavior of the metrics handler is configurable on a per-test basis. See [examples/metrics](examples/metrics) for some example config files, which are written in JSON. These are loaded into your test config via the `METRIC_CONFIG` key as shown in [examples/pt-cifar-tpu-v3-8.yaml](examples/pt-cifar-tpu-v3-8.yaml)
```

Comment:
Links should be relative to the current file on GitHub, i.e. `[../examples/metrics]`